wandb/run_manager.py

```diff
@@ -798,6 +802,22 @@ def wrap_existing_process(self, pid, stdout_read_fd, stderr_read_fd, port=None):
 
         self._sync_etc(headless=True)
 
+    def _check_update_available(self):
+        try:
+            data = requests.get('https://pypi.org/pypi/wandb/json').json()
```

Comment:
Not sure what the default timeout is, we should probably add one of 1 or 2 seconds.